### PR TITLE
IOT stack docker compose

### PR DESCRIPTION
**Pull request by hedi-sfaxi**

### CODE REVIEW

File: raspberry/services/influxdb/build_settings.yml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+version: '1'
+application: IOTstack
```
Comment (dkgmgo) [inline raspberry/services/influxdb/build_settings.yml]: maybe change the name of the application

File: raspberry/services/docker-compose.save.yml
Comment (augusto-moraes) [inline raspberry/services/docker-compose.save.yml:1]: is it the same as docker-compose.yml ?
I don't think this one is really necessary

File: raspberry/services/nodered/addons_list.yml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+version: '1'
+application: IOTstack
```
Comment (augusto-moraes) [inline raspberry/services/nodered/addons_list.yml]: Consider changing the mame of the application here too ?

File: raspberry/volumes/grafana/data/alerting/1/__default__.tmpl
Comment (dkgmgo) [inline raspberry/volumes/grafana/data/alerting/1/__default__.tmpl:1]: maybe we should delete this later

File: raspberry/volumes/portainer-ce/data/certs/cert.pem
Comment (dkgmgo) [inline raspberry/volumes/portainer-ce/data/certs/cert.pem:1]: I don't this we are suppose to git potainer's data cause it just for visualizing containers and it's not a real part of the project, so we should add 'raspberry/volumes/portainer-ce/' in the .gitignore

**Review by augusto-moraes — DISMISSED** [review]
just a few questions

File: raspberry/services/influxdb/build_settings.yml (hunk 1 above)
Comment (hedi-sfaxi) [reply raspberry/services/influxdb/build_settings.yml]: Done

**Review by dkgmgo — APPROVED** [review]

**Pull request merged**